### DISCUSSION TITLE
set up password, NOAUTH Authentication required

### DISCUSSION THREAD

**Discussion by Kossi-Francois** [encode/broadcaster#50] [Q&A]
Hello, and thank you for building this great module.
I would like to use it on heroku with the heroku-redis addon, but when I try to send a message with the websocket I have this error *asyncio_redis.exceptions.ErrorReply: NOAUTH Authentication required.*
and I think it's because I need to config the password but I don't know how to do it, maybe at initialization * broadcast = Broadcast (REDIS_URL, password = "xxx") *, but I don't know how to do it as there is no documentation on this.
Thank you in advance for your help

**Comment by ross-spencer** [comment]
+1 for this if it isn't available yet. I noticed a closed PR that doesn't seem merged: https://github.com/encode/broadcaster/pull/13 

What is the current thinking? 

**NB.** as for workarounds, is anyone having success using a URL constructed as so: `redis://<password>@localhost:6739`?

**Reply by osirvics** [reply]
Any solution to this? I get similar error when I try to connect with a password
